fbw-a32nx/src/systems/fmgc/src/flightplanning/new/DataManager.ts

```diff
@@ -52,7 +52,7 @@ type PbdWaypoint = {
     pbdDistance: NauticalMiles,
 }
 
-export type PilotWaypoint = LatLonWaypoint | PbxWaypoint | PbdWaypoint;
+export type PilotWaypoint = LatLonWaypoint | PbxWaypoint | PbdWaypoint | null;
```

Comment:
Why can this be null?

Reply:
You're right, this could also be handled via another way